mail_optional_follower_notifications/i18n/mail_optional_follower_notifications.pot

```diff
@@ -0,0 +1,42 @@
+# Translation of Odoo Server.
```

Comment:
POT file is not needed anymore, so please remove it to avoid its desynchronization.